presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlIntegrationSmokeTest.java

```diff
@@ -419,6 +419,22 @@ public void testNativeMultipleInClauses()
         execute("SELECT count(*) FROM tpch.orders WHERE " + longInClauses);
     }
 
+    @Test
+    public void testTimestampWithTimeZoneUnwrapCast()
```

Comment:
I think you can try adding the test to one or both `TestUnwrapCastInComparison` classes